src/main/webapp/app/exercises/text/assess/textblock-feedback-editor/textblock-feedback-editor.component.html

```diff
@@ -4,6 +4,16 @@
         <span [jhiTranslate]="'artemisApp.AssessmentType.' + feedback.type"></span>
     </span>
 
+    <span
+        class="badge bg-primary findOrigin"
```

Comment:
I think that a slight color change might be a good additional indicator that the question mark is clickable, but that's only a minor comment which might be achieved easily with the following suggestion _(I did not try it out)_

```suggestion
        class="badge btn-primary findOrigin"
```

Reply:
I changed the color slightly 👍